cibuildwheel/windows.py

```diff
@@ -20,9 +21,14 @@
 IS_RUNNING_ON_TRAVIS = os.environ.get('TRAVIS_OS_NAME') == 'windows'
 
 
-def get_python_path(config):
-    nuget_args = get_nuget_args(config)
-    return os.path.join(nuget_args[-1], nuget_args[0] + "." + config.version, "tools")
+def get_python_path(config): 
+    if config.identifier.startswith('cp'):
+        nuget_args = get_nuget_args(config)
+        return os.path.join(nuget_args[-1], nuget_args[0] + "." + config.version, "tools")
+    elif config.identifier.startswith('pp'):
+        # Inside the PyPy zip file is a directory with the same name
+        filename = config.url.rsplit('/', 1)[-1]
+        return os.path.join("C:\\PyPy", os.path.splitext(filename)[0])
```

Comment:
It might be wise to have a dedicated `cibuildwheel` folder. In fact, the same apply for cpython but I missed that bit in the nuget PR.

Reply:
Ah, right; just curious: why specifically? Just to avoid the unexpected conflicts with existing files/installations?

And are you suggesting to add it here, just for PyPy or also for CPython, or should we make a separate PR after, to keep PRs clean?

Reply:
> why specifically? Just to avoid the unexpected conflicts with existing files/installations?

yes exactly

> And are you suggesting to add it here, just for PyPy or also for CPython, or should we make a separate PR after, to keep PRs clean?

Probably a separate PR for cpython then rebase this one. I might to this this weekend for cpython

Reply:
Great! :-)
It'll still take a while to figure out macOS, I think, so by that time, I can rebase

Reply:
#197 opened to change installed software folder.

Reply:
Done, thanks! :)